
package practica2exception;

public class Test {
    
    public static void main(String[] args) {
        
        int valor = 20;
        int desde = -525;
        int hasta = 10000;
        
        System.out.println("el numero "+valor+"..."+estaEnRango(int valor, int desde, int hasta));
        
        }
    }

